evansmusomi/react-native-102
# Shows and Deletes Places

*What this does*
- Adds ability to show place details
- Adds ability to delete selected place

*How to verify*
- Select a place on the find place list
- You should be navigated to a place detail screen
- Select the delete button to delete the place
- You should be navigated back to the find place screen and the place should be gone
